### ISSUE TITLE
[UI] Footer spacing could be reduced on marketing sites

### ISSUE THREAD

**Issue by drewstone** [webb-tools/webb-dapp#1246]
**Is your UI update related to a problem? Please describe.**
Lot of space in footer.

**Describe the solution you'd like (include image or link to prototype)**
Remove some of the space.

<img width="1721" alt="image" src="https://github.com/webb-tools/webb-dapp/assets/13153687/6ce7314d-bccb-4654-8b23-a1092c3535f5">

<img width="1719" alt="image" src="https://github.com/webb-tools/webb-dapp/assets/13153687/b7b69704-f5a1-427a-9200-18e455232c3d">

**Comment by drewstone** [comment]
Also `Built by Webb Foundation` seems poorly placed?

**Comment by vutuanlinh2k2** [comment]
@monaiuu can you check back the design of the footer for the marketing site, I think this task can be done in a short amount of time.

**Comment by monaiuu** [comment]
To address this issue, let's remove the excessive bottom padding of `158px` as shown below on both Webbsite & Tangle:
<img width="1929" alt="Screenshot 2023-06-06 at 10 32 59 AM" src="https://github.com/webb-tools/webb-dapp/assets/109539270/b7323d4d-c6aa-4aea-b286-ba7fdc8ab11d">

On the Figma design, we have a simplified footer component for the Tangle Site (see below); if we want to keep it consistent with Webbsite, **let's remove the text 'Built by Webb Foundation'** altogether since we do have '© 2023 Webb Technologies, Inc. All rights reserved.' in the copy of the footer. 
<img width="1293" alt="Screenshot 2023-06-06 at 10 41 10 AM" src="https://github.com/webb-tools/webb-dapp/assets/109539270/6adea011-fa56-4a2a-9318-238d3fc594d3">


### Proposed Tangle Footer
<img width="757" alt="Screenshot 2023-06-06 at 10 39 19 AM" src="https://github.com/webb-tools/webb-dapp/assets/109539270/594bc64f-7a96-4c24-b921-1f0502ea75b6">

**Comment by vutuanlinh2k2** [comment]
@monaiuu Yes I agree with **removing the Built by Webb Foundation** text! Let me create a quick PR for this!
